src/Cloud.js

```diff
@@ -59,23 +59,22 @@ export function run(
   return CoreManager.getCloudController().run(name, data, requestOptions);
 }
 
- /**
+/**
   * Gets data for the current set of cloud jobs.
   * @method getJobsData
   * @name Parse.Cloud.getJobsData
   * @param {Object} options
```

Comment:
There are a few places where you removed options parameter but they still exist in the docs. 

https://github.com/gajus/eslint-plugin-jsdoc

or 

https://eslint.org/docs/rules/valid-jsdoc

Maybe we can use something like these plugin to validate the docs.

Reply:
Right, gotta have a look.

Reply:
valid-jsdoc yields an additional 200 errors ;) let's add it later ;)

Reply:
I agree everything looks good to me here

Reply:
awesome!